client/lib/post-normalizer/rule-create-better-excerpt.js

```diff
@@ -70,12 +70,12 @@ export function formatExcerpt( content ) {
 	dom.id = '__better_excerpt__';
 
 	// strip any p's that are empty
-	toArray( dom.querySelectorAll( 'p' ) )
+	Array.from( dom.querySelectorAll( 'p' ) )
 		.filter( ( element ) => trim( element.textContent ).length === 0 )
 		.forEach( removeElement );
 
 	// remove styles for all p's that remain
-	toArray( dom.querySelectorAll( 'p' ) ).forEach( ( element ) => {
+	Array.from( dom.querySelectorAll( 'p' ) ).forEach( ( element ) => {
 		element.removeAttribute( 'style' );
 		element.removeAttribute( 'align' );
 	} );
```

Comment:
`forEach` is natively supported by `NodeList` and since CoreJS 3, we are polyfilling it automatically for IE11 (the fallback build). `Array.from` is not needed.

`for ( element of dom.querySelectorAll( 'p' ) )` should also work natively with auto-polyfills.

There are many existing `Array.from( nodeList )` usages and I don't think it's terribly important to remove them -- I was just curious what the support situation is and did a bit of research.

Reply:
I prefer leaving this one for consistency with what we're doing above, as `NodeList.filter()` isn't supported anyway.